### DISCUSSION TITLE
Just merge or resume merging two segments?

### DISCUSSION THREAD

**Discussion by rieje** [mifi/lossless-cut#1922] [Q&A] [answered]
I prefer to trim/merge on tmpfs (memory) but once in a while it runs out of space (how much space do I need--I think I need more space than just the sum of the size of all the segments that will be joined together, right?). Also I might want individual segments as separate files but then decide to want to join them losslessly. Is this possible with lossless-cut?

**Comment by mifi** [comment, marked as answer]
> how much space do I need--I think

about twice of that

> Also I might want individual segments as separate files but then decide to want to join them losslessly

yes. export as separate files, then drag them into losslesscut and select "concat/merge"

**Comment by CarAuto34** [comment]
Ty
